libraries/ESP8266HTTPClient/src/ESP8266HTTPClient.cpp

```diff
@@ -100,6 +104,7 @@ class BearSSLTraits : public TransportTraits
 protected:
     uint8_t _fingerprint[20];
 };
+#endif
```

Comment:
When wrapping a large block of code (especially more than one screeful) with `ifdef`, suggest adding a comment on the line with `#endif`, e.g. `#endif // KEEP_PRESENT_API`.

Reply:
Adding "// HTTPCLIENT_1_1_COMPATIBLE" 3 times